README.md

```diff
@@ -19,7 +19,7 @@ It must be added along with `log` to the project dependencies:
 ```toml
 [dependencies]
 log = "0.4.0"
-env_logger = "0.9.0"
+env_logger = "0.10.0"
 ```
```

Comment:
Can you instead change this to
```console
$ cargo add log env_logger
```

And if we do keep the current form, we should also update the release replacements in `Cargo.toml` so this happens automatically on release.

Reply:
Done.